ChangeLog

```diff
@@ -1,52 +1,65 @@
+	* Integration to Coq: ssrbool.v ssrfun.v and plugin.
+	  ssrtest also moved to Coq test suite.
+
+	* Library refactoring:
+	odd_order and real_closed: are now in separate repositories.
+
+	algC, complex, ssrnum: Library ssrnum.v now
+	provides an interface numClosedFieldType, which factors some
+	theory from both library complex.v (now in the real_closed
+	repository) and algC. In particular, Re, Im, 'i,
+	conjC, n.-root and sqrtC are now part of this generic
+	interface. In case of ambiguity, they can be casted to the type
+	algC, of complex algebraic numbers, via typing constraints. Some
+	theory was thus made more generic, and the corresponding lemmas,
+	previously defined in library algC.v (e.g. conjCK) now feature an
+	extra, non maximal implicit, parameter
+	(C : numClosedFieldType). This could break some
+	proofs. Additionally, a few ad hoc constructions in library
+	complex.v, e.g. -*+ or complex.Re, are now deprecated and should
+	be replaced by the corresponding ones provided by the
+	numClosedFieldType interface. The next revision might definitely
+	hide those constructions under a module.
+
+	Lemma dvdn_fact moved from prime to div
+
+	* Structures, changes in interfaces:
+	  numClosedFieldType
 
-	* Renamings or replacements:
+	* New Theorems:
+	iter_in, finv_in, inv_f_in, finv_inj_in, fconnect_sym_in,
+	iter_order_in, iter_finv_in, cycle_orbit_in, fpath_finv_in,
+	fpath_finv_f_in, fpath_f_finv_in
+	big_allpairs
+	uniqP, uniqPn
+	dec_factor_theorem,
+	mul_bin_down, mul_bin_left
+        abstract_context (now merged in Coq)
+
+	* Renamed/generalized:
 	  mul_Sm_binm -> mul_bin_diag
 	  divn1 -> divz1 (in intdiv)
-
-	* Generalized or extended:
-	  ltngtP, contra_eq, contra_neq, odd_opp
-
-	* Plugin:
-	  ssrpattern: compose nicely with Tactic Notation
-
-25/08/2016 - refactoring of algC and complex in ssrnum -
-	upcomming version 1.7
-	* ssrnum's interface numClosedFieldType factors some theory from
-	  both complex and algC. Because of that Re, Im, 'i, conjC, n.-root
-	  and sqrtC are not specialized to algC anymore. In case of
-	  ambiguity, they should be instanciated with algC using typing
-	  constraints. Moreoever all the lemmas from ssrnum that used to
-	  be in algC (like conjCK), now take an extra nonmaximal implicit
-	  argument (C : numClosedFieldType) which could break some proofs.
-	  Additionally, ad-hoc constructions from complex.v like -*+ or
-	  complex.Re are now deprecated and one should rely solely on the
-	  ssrnum interface. The next revision might definietly hide those
-	  constructions under a module.
-
-	* Structures that change:
-	  numClosedFieldType
-
-	* Renamed and generalized definitions:
 	  rootC -> nthroot
 	  algRe -> Re
 	  algIm -> Im
 	  algCi -> imaginaryC
-
-	* Renamed and generalized theorems:
+  	  reshape_index_leq -> reshape_leq
 	  Every theorem from ssrnum that used to be in algC
 
+	* Generalized or improved:
+	  ltngtP, contra_eq, contra_neq, odd_opp, nth_iota
+
+	* Plugin:
```

Comment:
I think these should go (we already have a line saying that the plugin is in Coq).

Reply:
Should it go somewhere in a Coq change log? It's bit fine-grained for the CHANGES.